docker-compose  not found in the test environment.

```
Run docker-compose -f docker-compose.yml up -d --build
/home/runner/work/ line 1: docker-compose: command not found
Error: Process completed with exit code 127.
```

--- a/.github/workflows/test.yaml
+++ b/.github/workflows/test.yaml
@@ -21,7 +21,7 @@ jobs:
         run: cp template.env .env
 
       - name: Build and start Docker containers
-        run: docker-compose -f docker-compose.yml up -d --build 
+        run: docker compose -f docker-compose.yml up -d --build 
 
       - name: Wait for Django to start
         run: |
@@ -48,4 +48,4 @@ jobs:
         run: docker exec k9archiver_django python manage.py test
 
       - name: Stop and remove Docker containers
-        run: docker-compose down
+        run: docker compose down

--- a/operation.sh
+++ b/operation.sh
@@ -19,19 +19,19 @@ mkdir -p "$BACKUP_DIR/$COMMON_BACKUP_SUBDIR"
 
 
 backup_database() {
-    sudo docker exec "$POSTGRES_CONTAINER_NAME" pg_dumpall -U "$POSTGRES_USER"  > "$BACKUP_DIR/$COMMON_BACKUP_SUBDIR/$(date +'%Y%m%d%H%M%S')_postgres_backup.backup"
+    docker exec "$POSTGRES_CONTAINER_NAME" pg_dumpall -U "$POSTGRES_USER"  > "$BACKUP_DIR/$COMMON_BACKUP_SUBDIR/$(date +'%Y%m%d%H%M%S')_postgres_backup.backup"
 }
 
 backup_media() {
-    sudo docker exec "$MEDIA_CONTAINER_NAME" tar czf - /media/ > "$BACKUP_DIR/$COMMON_BACKUP_SUBDIR/$(date +'%Y%m%d%H%M%S')_media_backup.tar.gz"
+    docker exec "$MEDIA_CONTAINER_NAME" tar czf - /media/ > "$BACKUP_DIR/$COMMON_BACKUP_SUBDIR/$(date +'%Y%m%d%H%M%S')_media_backup.tar.gz"
 }
 
 
 # Deploy function
 deploy() {
     git pull origin master
-    sudo docker-compose down
-    sudo docker-compose -f docker-compose.prod.yml up -d --build
+    docker compose down
+    docker compose -f docker-compose.prod.yml up -d --build
 }
 
 # Check command-line argument

